sklearn_numba_dpex/common/kernels.py

```diff
@@ -135,7 +142,7 @@ def make_broadcast_ops_1d_2d_axis1_kernel(size0, size1, ops, work_group_size):
     """
 
     global_size = math.ceil(size1 / work_group_size) * work_group_size
-    ops = dpex.func(ops)
+    ops = dpex.func(ops())
```

Comment:
Similarly, is it normal to call `ops`, here? Shouldn't we just pass is as an argument? If that's a workaround, can you document it (as temporarily)?